packages/xo-web/src/common/combobox.js

```diff
@@ -30,7 +31,7 @@ export default class Combobox extends Component {
 
     props.className = 'form-control'
     props.onChange = this._handleChange
-    const Input = <input {...props} />
+    const Input = props.multiline ? <textarea {...props} /> : <input {...props} />
```

Comment:
I can't put a comment on line 30, but I think you should put a default value  `false`  to multiline, that way, you won't have to handle undefined later on